eslint.config.ts

```diff
@@ -0,0 +1,22 @@
+import tsParser from "@typescript-eslint/parser";
+import tsPlugin from "@typescript-eslint/eslint-plugin";
+import prettierConfig from "eslint-plugin-prettier/recommended";
+
```

Comment:
should add a codecv yaml to only check the src folder

Reply:
That will helps.